sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -984,6 +993,20 @@ public partial class HttpPipelineTransportOptions
         public System.Collections.Generic.IList<System.Security.Cryptography.X509Certificates.X509Certificate2> ClientCertificates { get { throw null; } }
         public System.Func<Azure.Core.Pipeline.ServerCertificateCustomValidationArgs, bool>? ServerCertificateCustomValidationCallback { get { throw null; } set { } }
     }
+    public abstract partial class RetryPolicy : Azure.Core.Pipeline.HttpPipelinePolicy
```

Comment:
This may be a weird Anne thing, but I'm not used to seeing an abstract class without an abstract method - i.e. where it is clear from the contract what the caller is expected to implement in the base/concrete class.  Do we have expectations regarding what customers would want to override, or are we just saying "here are some extensibility points, you can use any of them"?

Reply:
> or are we just saying "here are some extensibility points, you can use any of them"?

Yep!

Reply:
Here is an example of another abstract policy with no abstract methods - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Pipeline/HttpPipelineSynchronousPolicy.cs